recipes/mimalloc/0002-fix-install-path.patch

```diff
@@ -0,0 +1,57 @@
+From 240debabb57f3b13be9b2e287a39da79eaa77fc5 Mon Sep 17 00:00:00 2001
+From: elbaro <elbaro@users.noreply.github.com>
+Date: Tue, 14 Apr 2020 20:30:42 +0900
+Subject: [PATCH] fix install path
+
+---
+ CMakeLists.txt | 13 +++++--------
+ 1 file changed, 5 insertions(+), 8 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e16830a..6afa0b5 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -146,7 +146,7 @@ endif()
+ # Install and output names
+ # -----------------------------------------------------------------------------
+ 
+-set(mi_install_dir "${CMAKE_INSTALL_PREFIX}/lib/mimalloc-${mi_version}")
++set(mi_install_dir "${CMAKE_INSTALL_PREFIX}")
```

Comment:
Can you add a `lib/` at the end?

Reply:
That will install under lib/lib.

The current mimalloc realease has a problem with installing path:
https://github.com/microsoft/mimalloc/issues/223

It installs `.o` and `.h` under a wrong path, e.g. `$PREFIX/lib/mimalloc-1.6.1/include/mimalloc_xx.h`

See this test: https://github.com/conda-forge/staged-recipes/pull/11194/files#diff-aee5d9e631c2be348a8e340de9d64b30R37